src/test/java/com/fittracker/fittracker/controller/WeightControllerIntegrationTest.java

```diff
@@ -184,6 +189,10 @@ private void createUser() {
         User user = new User("user","user@example.com");
         user.setPassword("$2a$10$2gvLjc6wUEgM42M73tQ9ieI2jrAwfxap3X7XsEt//swQvJXyMpVJ6");
         User dbUser = userRepository.save(user);
-        testUUID = dbUser.getId();
+
+        LoginRequest loginRequest = new LoginRequest("user","password");
+        token = authenticationService.login(loginRequest).token();
```

Comment:
We could also make a request to /login endpoint, but this is ok for now. 👍 I have some idea for integration test improvement around that area, we can discuss it later.